mindsdb/integrations/libs/llm/utils.py

```diff
@@ -165,7 +165,7 @@ def get_llm_config(provider: str, config: Dict) -> BaseLLMConfig:
         )
     if provider == 'mindsdb':
         return MindsdbConfig(
-            llm_model_name=config['model_name'],
+            model_name=config['model_name'],
```

Comment:
Out of curiosity, why are we renaming this parameter? I may be missing something, but it looks like we don't need to rename it.

Reply:
Tyler and I thought it would be easier to keep it consistent with the naming of the other models

Reply:
Got it, makes sense!